keps/sig-node/2570-memory-qos/README.md

```diff
@@ -0,0 +1,121 @@
+# KEP-2570: Support Memory QoS using cgroups v2
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+  - [Comparison with Memory Manager](#comparison-with-memory-manager)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Feature Gate](#feature-gate)
+  - [Rules](#rules)
+  - [Interactive](#interactive)
+  - [Workflow](#workflow)
+    - [Guaranteed Pod](#guaranteed-pod)
+    - [Burstable Pod](#burstable-pod)
+  - [CGroups v2 Support](#cgroups-v2-support)
+  - [CGroup Hierarchy](#cgroup-hierarchy)
+  - [Container Runtime Interface (CRI) Changes](#container-runtime-interface-cri-changes)
+  - [Test Plan](#test-plan)
+<!-- /toc -->
+## Summary
+Support memory qos in pod qos class using cgroups v2.
+
+## Motivation
+In traditional cgroups v1 implement in Kubernetes, pod qos class and qos manager can only limit cpu resources, such as cpu_shares, memory qos is not yet implemented. cgroups v2 brings new capabilities for memory controller and it would help Kubernetes enhance memory isolation quality.
+
+### Goals
```

Comment:
Maybe add here under goals something along the lines of:

* Provide more strong guarantees around memory availability for guaranteed pods 
* Provide (best effort) guarantees around memory availability for burstable pods 
* Make use of new cgroup v2 memory controller settings such as `memory.min`, `memory.low` for pod and container cgroups`

Reply:
Thanks for suggestion. Have updated.